### PR TITLE
Add more UoM and associated values to measure tool

### PR DESCRIPTION
**Pull request by jtarricone**
Adds metric equivalent measurements to the output of the measuring tool, for length and area. Also adds acres/hectares to the area measurement.

To test:
* clone this branch and open the app
* activate the measuring tool
* when measuring, check that the metric equivalent value is displayed and is accurate
* check that the segment- and total-length measurements look correct, i.e. the total >= segment
* create a polygon and check that the area is now shown in square miles/kilometers and acres/hectares

Connects #784

### CODE REVIEW

File: src/GeositeFramework/js/widgets/map_utils/measure/AgsMeasure.js [hunk 1]
```diff
@@ -210,14 +211,32 @@ define([
                     dojo.connect(_pointLayer, 'onDblClick', handleMarkerDblClick);
             },
 
+            getLength = function(line) {
+                return _.map(options.esriLengthUnits, function (u) {
```
Comment (mmcfarland) [inline src/GeositeFramework/js/widgets/map_utils/measure/AgsMeasure.js]: I presume `u` is representing a unit variable, it would be easier to read and comprehend with a full variable name like `unit`.

File: src/GeositeFramework/js/widgets/map_utils/measure/AgsMeasure.js [hunk 2]
```diff
@@ -261,10 +284,21 @@ define([
 
                 // Calculate the length of the line, using a geographic coordindate system
                 geographicLine = webMercatorUtils.webMercatorToGeographic(line);
-                var geoLineLength = geodesicUtils.geodesicLengths([geographicLine], options.esriLengthUnits)[0];
+                var geoLineLength = getLength(geographicLine);
 
                 // Format the segment and line lengths
-                tipText = formatTooltip(geoLineLength, _renderedLength + geoLineLength);
+                var totalLength = _.map(
```
Comment (mmcfarland) [inline src/GeositeFramework/js/widgets/map_utils/measure/AgsMeasure.js]: I find this block pretty terse and hard to follow.  Breaking it down to component parts is about equivilent length but much better to read, in my opinion.  Along the lines of
```js
                var units = _.map(options.esriLengthUnits, function(unit) {
                        return _unitsLookup[unit];
                    }),
                    allSegments = _renderedLength.concat(geoLineLength),
                    totalLength = _.map(units, function(unit) {
                        return {
                            uom: unit,
                            length: sum(_.where(allSegments, { uom: unit }))
                        };
                    });
                function sum(arr) {
                    return _.reduce(arr, function (x, y) { return x + y.length; }, 0)
                }
```

File: src/GeositeFramework/Views/Home/Index.cshtml [hunk 3]
```diff
@@ -554,25 +554,44 @@
...
 
     <script type="text/template" id="template-measure-infobubble">
         <div class="measure-bubble">
-            <% if (area) { %>
+            <% if (areas) { %>
                 <p>
                     <span class="measure-label i18n" data-i18n="Area:">Area:</span>
-                    <span><%= area %> <%= areaUnits%><sup>2</sup></span>
+                    <div>
+                        <% _.each(areas, function (item, index) { %>
```
Comment (mmcfarland) [inline src/GeositeFramework/Views/Home/Index.cshtml]: The results look correct, but they're hard to read in that popup and the delete button is obscured.  Perhaps we should display them stacked vertically and just drop the "length" for area, which I assume means perimeter (if they want to measure lenght, let them draw a line).  Then we could move the delete button up to the right side, or at least make the popup taller so that it's visible.

Comment (jtarricone) [inline src/GeositeFramework/Views/Home/Index.cshtml]: I left the styling alone since I didn't see any wireframes or other guidance for how the box should be laid out. I thought that  a "table" with two columns, one for imperial and one for metric, might be a decent layout?

Comment (jtarricone) [inline src/GeositeFramework/Views/Home/Index.cshtml]: I looked into updating the tooltip, and it looks like it involves modifying a couple CSS files, possibly in addition to a code change somewhere. The `height` rule on the tooltip itself isn't enough to fix the problem, since there are rules for the container `div` that also have to be updated. That `div` is within the dijit info window, and it's dimension appear to be specific to the element rather than derived from a specific stylesheet, so it seemed like resizing the box might involve enough untangling to be handled separately.

File: src/GeositeFramework/js/widgets/map_utils/measure/AgsMeasure.js (hunk 2 above)
Comment (jtarricone) [inline src/GeositeFramework/js/widgets/map_utils/measure/AgsMeasure.js]: I refactored this function bit by splitting the functions and values into separate pieces to make the sequence a bit clearer.
